java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -252,9 +252,10 @@ public void set_returns_success() {
         // setup
         String key = "testKey";
         String value = "testValue";
-        CompletableFuture<Void> testResponse = mock(CompletableFuture.class);
-        when(testResponse.get()).thenReturn(null);
-        when(commandManager.<Void>submitNewCommand(eq(SetString), eq(new String[] {key, value}), any()))
+        CompletableFuture<String> testResponse = new CompletableFuture<>();
+        testResponse.complete(null);
+        when(commandManager.<String>submitNewCommand(
```

Comment:
our tests do almost nothing if they pass with such mistake

Reply:
Haha.  I noticed.  It doesn't care about the typing.  Really, we should be failing with a warning... otherwise these tests are just checking that the right argument/requestType is being passed to protobuf.